src/aiai_eval/config.py

```diff
@@ -160,8 +166,35 @@ class EvaluationConfig:
     verbose: bool
     track_carbon_emissions: bool
     country_iso_code: str
+    prefer_mps: bool
+    prefer_cpu: bool
     testing: bool = False
 
+    @property
+    def device(self) -> str:
+        """The compute device to use for the evaluation.
+
+        Returns:
+            str:
+                The compute device to use for the evaluation.
+        """
+
```

Comment:
```suggestion
if self.prefer_cpu and self.prefer_mps:
   raise InvalidEvaluation("Both '--prefer-cpu' and '--prefer-mps' where enabled.")
```

Reply:
I think it a bit unclear (at least from the users perspective) what it means to have both of these enabled? Maybe? 
Or keep the current behaviour, but throw a warning, instead?

Reply:
I changed it now to `prefer_device` instead, making it a bit more clear that these are mutually exclusive